.github/workflows/local_integration_tests.yml

```diff
@@ -9,11 +9,18 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
-    name: Run local integration tests
     strategy:
       fail-fast: false
-
+      matrix:
+        python:
+          - 3.8 # (almost) oldest Python supported by PSF
+          - 3.10 # newest Python that is stable
```

Comment:
I think `3.10` needs to be in quotes so that it doesn't get converted to decimal.